### PR TITLE
Impr Web API functional testing setup Instructions

### PR DESCRIPTION
**Pull request by diazwatson**
## Purpose of this pull request

This pull request (PR) makes more clear the instructions to setup Web API functional testing

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/get-started/web-api-functional-testing.html

## Links to Magento source code

- ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/get-started/web-api-functional-testing.md [hunk 1]
```diff
@@ -103,6 +103,6 @@ Now you are ready to run the tests.
 
 3. Copy `/dev/tests/api-functional/config/install-config-mysql.php.dist` to `/dev/tests/api-functional/config/install-config-mysql.php`.
 
-4. Configure your DB connection and install settings in `/dev/tests/api-functional/config/install-config-mysql.php`. Specify the Magento database. The base URL to access this Magento instance must be specified copied file i.e. `phpunit_rest.xml` or `phpunit_soap.xml`.
+4. Configure your DB connection and install settings in `/dev/tests/api-functional/config/install-config-mysql.php`. Specify the Magento database. The base URL to access this Magento instance must be the same specified in file `phpunit_rest.xml` or `phpunit_soap.xml`.
```
Comment (keharper) [inline guides/v2.3/get-started/web-api-functional-testing.md]:
```suggestion
4. Configure your DB connection and install settings in `/dev/tests/api-functional/config/install-config-mysql.php`. Specify the Magento database. The base URL to access this Magento instance must be the same specified in the `phpunit_rest.xml` or `phpunit_soap.xml` file.
```

**Comment by diazwatson** [comment]
Makes sense, thanks @keharper

**Comment by keharper** [comment]
Thanks @diazwatson

**Pull request merged**